### PR TITLE
ng_ipv6: set interface for multicast

### PR DESCRIPTION
**Pull request by OlegHahm**
If no interface is given for a multicast, but the node has only one, it must be set.

### CODE REVIEW

File: sys/net/network_layer/ng_ipv6/ng_ipv6.c [hunk 1]
```diff
@@ -298,12 +298,12 @@ static void _send_multicast(kernel_pid_t iface, ng_pktsnip_t *pkt,
 {
     ng_pktsnip_t *netif;
```
Comment (authmillenon) [inline sys/net/network_layer/ng_ipv6/ng_ipv6.c]: This newline is weird now.

**Comment by authmillenon** [comment]
ACK apart from my comment

**Comment by OlegHahm** [comment]
addressed and squashed

File: sys/net/network_layer/ng_ipv6/ng_ipv6.c [hunk 2]
```diff
@@ -297,13 +297,12 @@ static void _send_multicast(kernel_pid_t iface, ng_pktsnip_t *pkt,
                             bool prep_hdr)
 {
     ng_pktsnip_t *netif;
+    kernel_pid_t *ifs;
 
-#if NG_NETIF_NUMOF > 1
-    /* netif header not present: send over all interfaces */
     if (iface == KERNEL_PID_UNDEF) {
         size_t ifnum;
```
Comment (authmillenon) [inline sys/net/network_layer/ng_ipv6/ng_ipv6.c]: This needs to be moved out of this block, otherwise l320 will yield an error on NG_NETIF_NUMOF > 1.

**Comment by authmillenon** [comment]
re-ACK please squash

**Comment by authmillenon** [comment]
oh to slow :D

**Comment by OlegHahm** [comment]
done

**Pull request merged**